src/Sylius/Component/Inventory/tests/Checker/AvailabilityCheckerTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\Component\Inventory\Checker;
+
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Sylius\Component\Inventory\Checker\AvailabilityChecker;
+use Sylius\Component\Inventory\Checker\AvailabilityCheckerInterface;
+use Sylius\Component\Inventory\Model\StockableInterface;
+
+final class AvailabilityCheckerTest extends TestCase
+{
+    private AvailabilityChecker $availabilityChecker;
+
+    /** @var StockableInterface&MockObject */
+    private StockableInterface $stockable;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->availabilityChecker = new AvailabilityChecker();
+        $this->stockable = $this->createMock(StockableInterface::class);
+    }
+
+    public function testShouldImplementAvailabilityCheckerInterface(): void
+    {
+        self::assertInstanceOf(AvailabilityCheckerInterface::class, $this->availabilityChecker);
+    }
+
+    public function testRecognizeStockableAsAvailableIfOnHandQuantityIsGreaterThan0(): void
+    {
+        $this->stockable->method('isTracked')->willReturn(true);
```

Comment:
```suggestion
        $this->stockable->expects(self::once())->method('isTracked')->willReturn(true);
```
This class is missing the method invocation expectation. Because right now, you're not asserting that any of the methods on the `stockable` object have been called.

Reply:
fixed, thank You!